h/app.py

```diff
@@ -42,6 +22,28 @@ def create_app(global_config, **settings):
 
     config.add_subscriber('h.subscribers.add_renderer_globals',
                           'pyramid.events.BeforeRender')
+    config.add_subscriber('h.subscribers.set_user_from_oauth',
+                          'pyramid.events.NewRequest')
+
+    config.add_tween('h.tweens.csrf_tween_factory')
+    config.add_tween('h.tweens.auth_token')
+
+    config.include(__name__)
+
+    app = config.make_wsgi_app()
+    app = permit_cors(app,
```

Comment:
In fact our assets are explicitly listed as `Access-Control-Allow-Origin: *` independently of `permit_cors` usage.